### PR TITLE
Adds postal_code column

### PR DESCRIPTION
**Pull request by rapala61**
#### What's this PR do?
Adds postal_code column

#### What are the relevant tickets?
https://www.pivotaltracker.com/story/show/168433833

### CODE REVIEW

File: data/sql/derived-tables/campaign_activity.sql [hunk 1]
```diff
@@ -112,6 +112,7 @@ CREATE MATERIALIZED VIEW public.posts AS
 	    	 ELSE NULL END AS is_accepted,
 	    pd.action_id,
 	    pd.location,
+		pd.postal_code,
 	    a.reportback AS is_reportback,
```
Comment (blisteringherb) [inline data/sql/derived-tables/campaign_activity.sql]: I think the indentation might be off here?

File: quasar/dbt/models/campaign_activity/posts.sql [hunk 2]
```diff
@@ -36,6 +36,7 @@ SELECT
     	 ELSE NULL END AS is_accepted,
     pd.action_id,
     pd.location,
+	pd.postal_code,
```
Comment (blisteringherb) [inline quasar/dbt/models/campaign_activity/posts.sql]: Indentation?

File: data/sql/derived-tables/campaign_activity.sql (hunk 1 above)
Comment (rapala61) [reply data/sql/derived-tables/campaign_activity.sql]: the file has a mix of tabs and spaces. That line is using tabs (2). 🤔

Comment (rapala61) [reply data/sql/derived-tables/campaign_activity.sql]: Fixed the whole file to use tabs (4 spaced) only.

**Pull request merged**